src/facade/command_id.h

```diff
@@ -93,7 +93,7 @@ class CommandId {
   static uint32_t OptCount(uint32_t mask);
 
  protected:
-  std::string_view name_;
+  std::string name_;
```

Comment:
absl can return string_views when splitting so it was safe to keep it like this

Reply:
I know, but I accidentally tried to insert a string, which would then go out of scope and crash. I am not that happy of having non owning views for a class that's suppose to own it's members (that is, string instead of string view).